data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostAttesterSlashing.java

```diff
@@ -26,29 +26,31 @@
 import tech.pegasys.teku.infrastructure.restapi.endpoints.EndpointMetadata;
 import tech.pegasys.teku.infrastructure.restapi.endpoints.RestApiEndpoint;
 import tech.pegasys.teku.infrastructure.restapi.endpoints.RestApiRequest;
-import tech.pegasys.teku.spec.Spec;
+import tech.pegasys.teku.spec.SpecMilestone;
 import tech.pegasys.teku.spec.datastructures.operations.AttesterSlashing;
-import tech.pegasys.teku.spec.datastructures.operations.IndexedAttestation;
+import tech.pegasys.teku.spec.schemas.SchemaDefinitionCache;
 import tech.pegasys.teku.statetransition.validation.InternalValidationResult;
 import tech.pegasys.teku.statetransition.validation.ValidationResultCode;
 
 public class PostAttesterSlashing extends RestApiEndpoint {
   public static final String ROUTE = "/eth/v1/beacon/pool/attester_slashings";
   private final NodeDataProvider nodeDataProvider;
 
-  public PostAttesterSlashing(final DataProvider dataProvider, final Spec spec) {
-    this(dataProvider.getNodeDataProvider(), spec);
+  public PostAttesterSlashing(
+      final DataProvider dataProvider, final SchemaDefinitionCache schemaDefinitionCache) {
+    this(dataProvider.getNodeDataProvider(), schemaDefinitionCache);
   }
 
-  public PostAttesterSlashing(final NodeDataProvider provider, final Spec spec) {
+  public PostAttesterSlashing(
+      final NodeDataProvider provider, final SchemaDefinitionCache schemaDefinitionCache) {
```

Comment:
Actually we don't need the Spec anymore since we know this API supports Pre Electra attester slashing only so we can't directly get the schema from the schema definition cache specifying the `PHASE0` mile stone. So yes, this could also be applied to the `PostAttesterSlashings` API as well, check [this PostAttestationV2 API PR](https://github.com/Consensys/teku/pull/8448) for reference